### PR TITLE
Train simple NN

### PR DESCRIPTION
**Pull request by grahamsg**
The notebook heat_eqn.ipynb contains an attempt to train a simple neural network, with no physics term added to the loss function.

### CODE REVIEW

**Comment by anandk97** [comment]
Merging pull request to take a look at Graham's code, already reviewed your Github for the assignment.
